packages/sitecore-jss-nextjs/src/services/graphql-sitemap-service.ts

```diff
@@ -0,0 +1,252 @@
+import { GraphQLRequestClient } from '@sitecore-jss/sitecore-jss';
+
+export type GraphQLSitemapServiceConfig = {
+  /**
+   * Your Graphql endpoint
+   */
+  endpoint: string;
+};
+
+type StaticPath = {
+  params: {
+    path: string[];
+  };
+  locale?: string;
+};
+
+type SearchResult = {
+  search: {
+    results: { url: { path: string } }[];
+  };
+};
+
+export class GraphQLSitemapService {
+  /**
+   * Provides ability to fetch sitemap from graphql endpoint.
+   * Sitemap can be used for SSG/Export
+   * @param {GraphQLSitemapServiceConfig} config graphql sitemap service config
+   */
+  constructor(private config: GraphQLSitemapServiceConfig) {}
+
+  /**
+   * Fetch sitemap which could be used for generation of static pages using Export mode
```

Comment:
We might want to call out the difference w/r/t locale handling between these two. e.g.
"Fetch sitemap which could be used for generation of static pages during 'next export'. The 'locale' parameter will be used in the item query, but since i18n is not supported, the output paths will not include a 'locale' property."

Reply:
Thank you, changed